TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift

```diff
@@ -181,8 +183,10 @@ public class TwilioVerifyBuilder {
     self.clearStorageOnReinstall = clearStorageOnReinstall
     return self
   }
-  
-  func setURL(_ url: String) -> Self {
+
+  /// Set the baseURL that will be used by the NetworkProvider.
+  /// - Parameter url: String base`URL`.
+  public func setBaseURL(_ url: String) -> Self {
```

Comment:
Why do we need to change this method to public?

Reply:
From my point of view, make public the setter for the network provider won't fix the constraint related to a specific domain, since the network provider receives the requests with the URL already created and then execute them. So I think the baseURL would be necessary to make it work properly. But I'm not sure, what do you think @yafuquen ?

Reply:
The network provider can change the base url when receiving the request. Making the set url method public could create a backdoor

Reply:
@yafuquen Given that concern, what do you think about changing the setter of the `NetworkProvider` to a parameter in the initializer? This should prevent any further modification of the object after being initialized if for some reason is intercepted.

Reply:
This is a builder, so there is not a problem changing it after being initialized, because with the builder you will get the SDK instance, the `build` method will get the network provider to use (the one created in the init method or a new one passed in the set method) and it will depend on the developer implementation

Reply:
Yeah, I think these security concerns apply for both the `NetworkProvider` & the `baseURL`, but it would depend on the developer implementation, so probably changing only the setter for `NetworkProvider` would be the best right now. I'm rolling back the change for the `baseURL` 👍